src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -288,7 +273,7 @@
 
                               collectible
                               (str (:name collection-data) " #" collectible-id))
-         owner-address      (-> db :wallet :current-viewing-account-address)
+         owner-address      (-> collectible :ownership first :address)
```

Comment:
Could you please clarify the reason of this change?
As far as I remember the `:ownership` field contains addresses of all accounts that own collectible. Taking the `first` address will make it impossible to send multi-collectible from all others.

Reply:
We are currently getting the owner-address from `(-> db :wallet :current-viewing-account-address)`, so that won't work for multi-collectibles either. If the user starts the flow from the general wallet screen, this value may be `nil` or some other address that is not any owner of the collectible. So, this is just a patch to at least avoid errors when fetching routes in that case. @smohamedjavid is working on proper fixes for multi-collectibles as part of #21727, as this exceeds this PR scope. Not a big deal for now, I can rollback this change and wait for Javid's fixes!

Reply:
@briansztamfater I think it is better to merge your PR without this change. Because if we won't change anything here - the error will be more obvious. But with your change the error becomes more "hidden", because it works almost correct :)) I also don't think that it should step the PR from merging, especially if the fix is coming.